src/main/resources/assets/securitycraft/lang/en_us.json

```diff
@@ -370,53 +370,53 @@
+	"help.securitycraft.keypad_frame.info": "The frame is used to craft keypads. Rightclick it with a key panel to craft a keypad. Views automatically shutoff after 5 minutes to save resources. (Rightclick again to turn it back on.)",
+	"help.securitycraft.claymore.info": "Claymores explode one second after a mob or player walks within 5 blocks in front of the mine. Rightclicking the claymore while holding wire cutters will defuse the mine and allow you to break it. Rightclicking with flint and steel equipped will re-enable it.",
 	"help.securitycraft.keypad_furnace.info": "The password-protected furnace will require you to enter a passcode when you first place it down. After that, entering the correct code into the GUI will allow you to access the furnace's inventory.",
-	"help.securitycraft.security_camera.info": "The security camera allows you to view the nearby area from its position. Simply place down a camera, then rightclick it with a camera monitor to use it.",
+	"help.securitycraft.security_camera.info": "The security camera allows you to view the nearby area from its position. Simply place down a camera, then rightclick it with a camera monitor to bind it for use.",
 	"help.securitycraft.reinforced_glass_pane.info": "The reinforced glass pane is the same as a vanilla glass, except only the person who placed it down can destroy it using the universal block remover.",
 	"help.securitycraft.reinforced_iron_bars.info": "The reinforced iron bars are the same as a vanilla iron bars, except only the person who placed them down can destroy them using the universal block remover.",
 	"help.securitycraft.electrified_iron_fence.info": "The electrified iron fence acts the same as a vanilla fence, except it is unbreakable, and will hurt anyone but the owner when they get in contact with it.",
-	"help.securitycraft.ims.info": "The I.M.S, or Intelligent Munitions System, is a four-time-use mine that will target players (or mobs) which enter its firing radius. Once a mine is launched, it will track down the entity and explode. Once all four mines are used, the I.M.S. will self-destruct.",
-	"help.securitycraft.protecto.info": "The protecto, inspired by the enemy from the game \"Megaman: Battle Network 2\" made by Capcom, will attack any mob in a 10-block radius with lightning bolts every 10 seconds. It must be raining, and able to see the sky to do so.",
+	"help.securitycraft.ims.info": "The I.M.S, or Intelligent Munitions System, is a refillable mine holding up to 4 Bouncing Betties that will target players or mobs which enter its firing radius. Once a mine is launched, it will track down the entity and explode. If mines are missing from the I.M.S., rightclick it with more Bouncing Betties to refill.",
```

Comment:
Which sentence?

Reply:
Well this one. The one that this comment is about.

Reply:
I think the new sentence is more accurate because the IMS doesn't self-destruct anymore and the old entry doesn't mention that the mines are refillable.